lib/zwave/system/capabilities/alarm_contact/NOTIFICATION.js

```diff
@@ -9,9 +9,17 @@ module.exports = {
 	}),
 	report: 'NOTIFICATION_REPORT',
 	reportParser: report => {
-		if (report && report.hasOwnProperty('Notification Type') && report['Notification Type'] === 'Access Control') {
+		if (report && report['Notification Type'] === 'Access Control Alarm' && report.hasOwnProperty('Event (Parsed)')) {
```

Comment:
Access Control -> Access Control Alarm, you sure the latter is correct?

Reply:
Woops :blush: fixed :+1: